code/ensemble/utils/run-ensemble.R

```diff
@@ -35,6 +35,7 @@ source(here("code", "config_utils", "get_hub_config.R"))
 run_ensemble <- function(method = "mean",
                          forecast_date,
                          exclude_models = NULL,
+                         exclude_designated_other = TRUE,
```

Comment:
Do you think it's worth having this as an explicit option here? An alternative would be to have `...` here and pass `...` to `use_ensemble_criteria` instead.